source/setup/setup.rst

```diff
@@ -2,69 +2,104 @@
 Cerebro Setup
 *************************
 
-Cerebro has a few core dependencies that need to be setup before it can be used.
+We provide a Dockerfile that installs all the dependencies required by Cerebro. Running the Dockerfile will build SCALE-MAMBA and EMP-AGMPC. 
+	* Create the Dockerfile and requirements.txt.
+	        * Dockerfile 
```

Comment:
Instead of putting the Dockerfile in the documentation, you can create the file and commit it to the repository. See an example here: https://github.com/opaque-systems/opaque/blob/master/docker/Dockerfile.